addons/web_editor/static/lib/odoo-editor/src/utils/utils.js

```diff
@@ -1370,7 +1370,7 @@ export function getOuid(node, optimize = false) {
  * @returns {boolean}
  */
 export function isHtmlContentSupported(node) {
-    return !closestElement(node, '[data-oe-model]:not([data-oe-field="arch"]),[data-oe-translation-id]', true);
+    return !closestElement(node, '[data-oe-model]:not([data-oe-field="arch"]):not([data-oe-type="html"]),[data-oe-translation-id]', true);
```

Comment:
About the commit message:
- It looks like there is a missing "the" at the beginning.
- Sometimes we forget, but we usually try to put code-related labels inside backticks like function names, html attributes or primitive values for exemple.
- Commit message should be wrapped at 80 characters.
- False is Python, in javascript it is false
- Personal taste: I'd rather have one sentence than three here. The fact that the previous behavior was incorrect can easily be woven inside the first sentence, and the fact that the commit fixes the incorrect behavior is obvious.

Considering all of the above, I suggest the following rewrite:
```
The `isHtmlContentSupported` function incorrectly returned `false` for elements
inside `data-oe-type="html"` nodes even though those in fact do support html.
```

Reply:
Changed.